python/lsst/dax/imgserv/api_soda.py

```diff
@@ -24,7 +24,7 @@
 Corresponding URI: /api/image/soda
 
 """
-import os
+import os, os.path
```

Comment:
I'm pretty sure this is a PEP8 violation. You need imports on separate lines. Did this pass flake8? It should fail with E401 error.

Reply:
yep, it was flagged by Pycharm.